src/load/azext_load/data_plane/utils/models.py

```diff
@@ -29,3 +31,8 @@ class AllowedIntervals(str, Enum):
 class AllowedMetricNamespaces(str, Enum):
     LoadTestRunMetrics = "LoadTestRunMetrics"
     EngineHealthMetrics = "EngineHealthMetrics"
+
+
+class AllowedTestTypes(str, Enum):
```

Comment:
I would suggest keeping a copy of its own client side, since any new test type might start appearing as allowed if we update the sdk but its corresponding changes are not done in cli

Reply:
Sure sounds fair